2025/05/08 08:50:19 [error] 216#216: *19 localhost could not be resolved (3: Host not found), client: 172.24.0.1, server: localhost, request: "GET / HTTP/1.1", host: "localhost", referrer: "http://localhost:81/"
2025/05/08 08:50:20 [error] 216#216: *19 localhost could not be resolved (3: Host not found), client: 172.24.0.1, server: localhost, request: "GET /favicon.ico HTTP/1.1", host: "localhost", referrer: "http://localhost/"
2025/05/08 08:50:21 [error] 216#216: *19 localhost could not be resolved (3: Host not found), client: 172.24.0.1, server: localhost, request: "GET / HTTP/1.1", host: "localhost", referrer: "http://localhost:81/"
2025/05/08 08:50:21 [error] 216#216: *19 localhost could not be resolved (3: Host not found), client: 172.24.0.1, server: localhost, request: "GET /favicon.ico HTTP/1.1", host: "localhost", referrer: "http://localhost/"
2025/05/08 08:50:23 [error] 216#216: *19 localhost could not be resolved (3: Host not found), client: 172.24.0.1, server: localhost, request: "GET / HTTP/1.1", host: "localhost", referrer: "http://localhost:81/"
2025/05/08 08:50:23 [error] 216#216: *19 localhost could not be resolved (3: Host not found), client: 172.24.0.1, server: localhost, request: "GET /favicon.ico HTTP/1.1", host: "localhost", referrer: "http://localhost/"
2025/05/08 08:56:06 [error] 238#238: *1539 localhost could not be resolved (3: Host not found), client: 172.24.0.1, server: testlocal, request: "GET / HTTP/1.1", host: "testlocal"
2025/05/08 08:56:06 [error] 238#238: *1539 localhost could not be resolved (3: Host not found), client: 172.24.0.1, server: testlocal, request: "GET /favicon.ico HTTP/1.1", host: "testlocal", referrer: "http://testlocal/"
2025/05/08 08:56:06 [error] 238#238: *1539 localhost could not be resolved (3: Host not found), client: 172.24.0.1, server: testlocal, request: "GET / HTTP/1.1", host: "testlocal"
2025/05/08 08:56:06 [error] 238#238: *1539 localhost could not be resolved (3: Host not found), client: 172.24.0.1, server: testlocal, request: "GET /favicon.ico HTTP/1.1", host: "testlocal", referrer: "http://testlocal/"
2025/05/08 08:56:06 [error] 238#238: *1539 localhost could not be resolved (3: Host not found), client: 172.24.0.1, server: testlocal, request: "GET / HTTP/1.1", host: "testlocal"
2025/05/08 08:56:07 [error] 238#238: *1539 localhost could not be resolved (3: Host not found), client: 172.24.0.1, server: testlocal, request: "GET /favicon.ico HTTP/1.1", host: "testlocal", referrer: "http://testlocal/"
2025/05/08 08:56:07 [error] 238#238: *1539 localhost could not be resolved (3: Host not found), client: 172.24.0.1, server: testlocal, request: "GET / HTTP/1.1", host: "testlocal"
2025/05/08 08:56:07 [error] 238#238: *1539 localhost could not be resolved (3: Host not found), client: 172.24.0.1, server: testlocal, request: "GET /favicon.ico HTTP/1.1", host: "testlocal", referrer: "http://testlocal/"
2025/05/08 08:56:07 [error] 238#238: *1539 localhost could not be resolved (3: Host not found), client: 172.24.0.1, server: testlocal, request: "GET / HTTP/1.1", host: "testlocal"
2025/05/08 08:56:07 [error] 238#238: *1539 localhost could not be resolved (3: Host not found), client: 172.24.0.1, server: testlocal, request: "GET /favicon.ico HTTP/1.1", host: "testlocal", referrer: "http://testlocal/"
2025/05/08 08:56:07 [error] 238#238: *1539 localhost could not be resolved (3: Host not found), client: 172.24.0.1, server: testlocal, request: "GET / HTTP/1.1", host: "testlocal"
2025/05/08 08:56:07 [error] 238#238: *1539 localhost could not be resolved (3: Host not found), client: 172.24.0.1, server: testlocal, request: "GET /favicon.ico HTTP/1.1", host: "testlocal", referrer: "http://testlocal/"
2025/05/08 08:56:07 [error] 238#238: *1539 localhost could not be resolved (3: Host not found), client: 172.24.0.1, server: testlocal, request: "GET / HTTP/1.1", host: "testlocal"
2025/05/08 08:56:07 [error] 238#238: *1539 localhost could not be resolved (3: Host not found), client: 172.24.0.1, server: testlocal, request: "GET /favicon.ico HTTP/1.1", host: "testlocal", referrer: "http://testlocal/"
2025/05/08 08:56:07 [error] 238#238: *1539 localhost could not be resolved (3: Host not found), client: 172.24.0.1, server: testlocal, request: "GET / HTTP/1.1", host: "testlocal"
2025/05/08 08:56:07 [error] 238#238: *1539 localhost could not be resolved (3: Host not found), client: 172.24.0.1, server: testlocal, request: "GET /favicon.ico HTTP/1.1", host: "testlocal", referrer: "http://testlocal/"
2025/05/08 08:56:07 [error] 238#238: *1539 localhost could not be resolved (3: Host not found), client: 172.24.0.1, server: testlocal, request: "GET / HTTP/1.1", host: "testlocal"
2025/05/08 08:56:07 [error] 238#238: *1539 localhost could not be resolved (3: Host not found), client: 172.24.0.1, server: testlocal, request: "GET /favicon.ico HTTP/1.1", host: "testlocal", referrer: "http://testlocal/"
2025/05/08 08:56:08 [error] 238#238: *1539 localhost could not be resolved (3: Host not found), client: 172.24.0.1, server: testlocal, request: "GET / HTTP/1.1", host: "testlocal"
2025/05/08 08:56:08 [error] 238#238: *1539 localhost could not be resolved (3: Host not found), client: 172.24.0.1, server: testlocal, request: "GET /favicon.ico HTTP/1.1", host: "testlocal", referrer: "http://testlocal/"
2025/05/08 08:56:08 [error] 238#238: *1539 localhost could not be resolved (3: Host not found), client: 172.24.0.1, server: testlocal, request: "GET / HTTP/1.1", host: "testlocal"
2025/05/08 08:56:08 [error] 238#238: *1539 localhost could not be resolved (3: Host not found), client: 172.24.0.1, server: testlocal, request: "GET / HTTP/1.1", host: "testlocal"
2025/05/08 08:56:08 [error] 238#238: *1539 localhost could not be resolved (3: Host not found), client: 172.24.0.1, server: testlocal, request: "GET /favicon.ico HTTP/1.1", host: "testlocal", referrer: "http://testlocal/"
2025/05/08 08:56:08 [error] 238#238: *1539 localhost could not be resolved (3: Host not found), client: 172.24.0.1, server: testlocal, request: "GET / HTTP/1.1", host: "testlocal"
2025/05/08 08:56:08 [error] 238#238: *1539 localhost could not be resolved (3: Host not found), client: 172.24.0.1, server: testlocal, request: "GET /favicon.ico HTTP/1.1", host: "testlocal", referrer: "http://testlocal/"
2025/05/08 08:56:08 [error] 238#238: *1539 localhost could not be resolved (3: Host not found), client: 172.24.0.1, server: testlocal, request: "GET / HTTP/1.1", host: "testlocal"
2025/05/08 08:56:08 [error] 238#238: *1539 localhost could not be resolved (3: Host not found), client: 172.24.0.1, server: testlocal, request: "GET /favicon.ico HTTP/1.1", host: "testlocal", referrer: "http://testlocal/"
2025/05/08 08:56:08 [error] 238#238: *1539 localhost could not be resolved (3: Host not found), client: 172.24.0.1, server: testlocal, request: "GET / HTTP/1.1", host: "testlocal"
2025/05/08 08:56:08 [error] 238#238: *1539 localhost could not be resolved (3: Host not found), client: 172.24.0.1, server: testlocal, request: "GET /favicon.ico HTTP/1.1", host: "testlocal", referrer: "http://testlocal/"
2025/05/08 08:56:08 [error] 238#238: *1539 localhost could not be resolved (3: Host not found), client: 172.24.0.1, server: testlocal, request: "GET / HTTP/1.1", host: "testlocal"
2025/05/08 08:56:08 [error] 238#238: *1539 localhost could not be resolved (3: Host not found), client: 172.24.0.1, server: testlocal, request: "GET /favicon.ico HTTP/1.1", host: "testlocal", referrer: "http://testlocal/"
2025/05/09 18:25:02 [error] 197#197: *1490 localhost could not be resolved (3: Host not found), client: 172.27.0.1, server: testlocal, request: "GET / HTTP/1.1", host: "testlocal"
2025/05/09 18:25:03 [error] 197#197: *1490 localhost could not be resolved (3: Host not found), client: 172.27.0.1, server: testlocal, request: "GET /favicon.ico HTTP/1.1", host: "testlocal", referrer: "http://testlocal/"
2025/05/09 18:25:04 [error] 197#197: *1490 localhost could not be resolved (3: Host not found), client: 172.27.0.1, server: testlocal, request: "GET / HTTP/1.1", host: "testlocal"
2025/05/09 18:25:04 [error] 197#197: *1490 localhost could not be resolved (3: Host not found), client: 172.27.0.1, server: testlocal, request: "GET /favicon.ico HTTP/1.1", host: "testlocal", referrer: "http://testlocal/"
2025/05/09 18:26:03 [error] 217#217: *1508 connect() failed (111: Connection refused) while connecting to upstream, client: 172.27.0.1, server: testlocal, request: "GET / HTTP/1.1", upstream: "http://172.27.0.7:5173/", host: "testlocal"
2025/05/09 18:26:03 [error] 217#217: *1508 connect() failed (111: Connection refused) while connecting to upstream, client: 172.27.0.1, server: testlocal, request: "GET /favicon.ico HTTP/1.1", upstream: "http://172.27.0.7:5173/favicon.ico", host: "testlocal", referrer: "http://testlocal/"
2025/05/09 18:26:04 [error] 217#217: *1508 connect() failed (111: Connection refused) while connecting to upstream, client: 172.27.0.1, server: testlocal, request: "GET / HTTP/1.1", upstream: "http://172.27.0.7:5173/", host: "testlocal"
2025/05/09 18:26:04 [error] 217#217: *1508 connect() failed (111: Connection refused) while connecting to upstream, client: 172.27.0.1, server: testlocal, request: "GET /favicon.ico HTTP/1.1", upstream: "http://172.27.0.7:5173/favicon.ico", host: "testlocal", referrer: "http://testlocal/"
2025/05/09 18:26:27 [error] 238#238: *1521 connect() failed (111: Connection refused) while connecting to upstream, client: 172.27.0.1, server: testlocal, request: "GET / HTTP/1.1", upstream: "http://172.27.0.7:80/", host: "testlocal", referrer: "http://testlocal:81/"
2025/05/09 18:26:27 [error] 238#238: *1521 connect() failed (111: Connection refused) while connecting to upstream, client: 172.27.0.1, server: testlocal, request: "GET /favicon.ico HTTP/1.1", upstream: "http://172.27.0.7:80/favicon.ico", host: "testlocal", referrer: "http://testlocal/"
2025/05/09 18:31:44 [error] 281#281: *1576 connect() failed (111: Connection refused) while connecting to upstream, client: 172.27.0.1, server: testlocal, request: "GET /api/users?page=1&limit=10&search= HTTP/1.1", upstream: "http://172.27.0.4:3000/api/users?page=1&limit=10&search=", host: "testlocal", referrer: "http://testlocal/users/list"
2025/05/09 18:31:49 [error] 281#281: *1576 connect() failed (111: Connection refused) while connecting to upstream, client: 172.27.0.1, server: testlocal, request: "GET /api/users?page=1&limit=10&search= HTTP/1.1", upstream: "http://172.27.0.4:3000/api/users?page=1&limit=10&search=", host: "testlocal", referrer: "http://testlocal/users/list"
2025/05/09 18:31:57 [error] 281#281: *1576 connect() failed (111: Connection refused) while connecting to upstream, client: 172.27.0.1, server: testlocal, request: "GET /api/users?page=1&limit=10&search= HTTP/1.1", upstream: "http://172.27.0.4:3000/api/users?page=1&limit=10&search=", host: "testlocal"
2025/05/09 18:31:59 [error] 281#281: *1576 connect() failed (111: Connection refused) while connecting to upstream, client: 172.27.0.1, server: testlocal, request: "GET /api/users?page=1&limit=10&search= HTTP/1.1", upstream: "http://172.27.0.4:3000/api/users?page=1&limit=10&search=", host: "testlocal"
2025/05/09 18:31:59 [error] 281#281: *1576 connect() failed (111: Connection refused) while connecting to upstream, client: 172.27.0.1, server: testlocal, request: "GET /api/users?page=1&limit=10&search= HTTP/1.1", upstream: "http://172.27.0.4:3000/api/users?page=1&limit=10&search=", host: "testlocal"
2025/05/09 18:32:00 [error] 281#281: *1576 connect() failed (111: Connection refused) while connecting to upstream, client: 172.27.0.1, server: testlocal, request: "GET /api/users?page=1&limit=10&search= HTTP/1.1", upstream: "http://172.27.0.4:3000/api/users?page=1&limit=10&search=", host: "testlocal", referrer: "http://testlocal/users/list"
2025/05/09 18:32:00 [error] 281#281: *1576 connect() failed (111: Connection refused) while connecting to upstream, client: 172.27.0.1, server: testlocal, request: "GET /api/users?page=1&limit=10&search= HTTP/1.1", upstream: "http://172.27.0.4:3000/api/users?page=1&limit=10&search=", host: "testlocal"
2025/05/09 18:32:00 [error] 281#281: *1576 connect() failed (111: Connection refused) while connecting to upstream, client: 172.27.0.1, server: testlocal, request: "GET /api/users?page=1&limit=10&search= HTTP/1.1", upstream: "http://172.27.0.4:3000/api/users?page=1&limit=10&search=", host: "testlocal"
2025/05/09 18:32:00 [error] 281#281: *1576 connect() failed (111: Connection refused) while connecting to upstream, client: 172.27.0.1, server: testlocal, request: "GET /api/users?page=1&limit=10&search= HTTP/1.1", upstream: "http://172.27.0.4:3000/api/users?page=1&limit=10&search=", host: "testlocal"
2025/05/09 18:32:00 [error] 281#281: *1576 connect() failed (111: Connection refused) while connecting to upstream, client: 172.27.0.1, server: testlocal, request: "GET /api/users?page=1&limit=10&search= HTTP/1.1", upstream: "http://172.27.0.4:3000/api/users?page=1&limit=10&search=", host: "testlocal"
2025/05/09 18:32:00 [error] 281#281: *1576 connect() failed (111: Connection refused) while connecting to upstream, client: 172.27.0.1, server: testlocal, request: "GET /api/users?page=1&limit=10&search= HTTP/1.1", upstream: "http://172.27.0.4:3000/api/users?page=1&limit=10&search=", host: "testlocal"
2025/05/09 18:32:06 [error] 281#281: *1576 connect() failed (111: Connection refused) while connecting to upstream, client: 172.27.0.1, server: testlocal, request: "GET /api/users?page=1&limit=10&search= HTTP/1.1", upstream: "http://172.27.0.4:3000/api/users?page=1&limit=10&search=", host: "testlocal", referrer: "http://testlocal/users/list"
2025/05/09 18:32:17 [error] 281#281: *1576 connect() failed (111: Connection refused) while connecting to upstream, client: 172.27.0.1, server: testlocal, request: "GET /api/users?page=1&limit=10&search= HTTP/1.1", upstream: "http://172.27.0.4:3000/api/users?page=1&limit=10&search=", host: "testlocal", referrer: "http://testlocal/users/list"
2025/05/09 18:35:28 [error] 282#282: *1611 connect() failed (111: Connection refused) while connecting to upstream, client: 172.27.0.1, server: testlocal, request: "GET /api/users?page=1&limit=10&search= HTTP/1.1", upstream: "http://172.27.0.4:3000/api/users?page=1&limit=10&search=", host: "testlocal"
2025/05/09 18:35:29 [error] 282#282: *1611 connect() failed (111: Connection refused) while connecting to upstream, client: 172.27.0.1, server: testlocal, request: "GET /api/users?page=1&limit=10&search= HTTP/1.1", upstream: "http://172.27.0.4:3000/api/users?page=1&limit=10&search=", host: "testlocal"
2025/05/09 18:35:29 [error] 282#282: *1611 connect() failed (111: Connection refused) while connecting to upstream, client: 172.27.0.1, server: testlocal, request: "GET /api/users?page=1&limit=10&search= HTTP/1.1", upstream: "http://172.27.0.4:3000/api/users?page=1&limit=10&search=", host: "testlocal"
2025/05/09 18:35:29 [error] 282#282: *1611 connect() failed (111: Connection refused) while connecting to upstream, client: 172.27.0.1, server: testlocal, request: "GET /api/users?page=1&limit=10&search= HTTP/1.1", upstream: "http://172.27.0.4:3000/api/users?page=1&limit=10&search=", host: "testlocal"
2025/05/09 18:35:29 [error] 282#282: *1611 connect() failed (111: Connection refused) while connecting to upstream, client: 172.27.0.1, server: testlocal, request: "GET /api/users?page=1&limit=10&search= HTTP/1.1", upstream: "http://172.27.0.4:3000/api/users?page=1&limit=10&search=", host: "testlocal"
2025/05/09 18:35:30 [error] 282#282: *1611 connect() failed (111: Connection refused) while connecting to upstream, client: 172.27.0.1, server: testlocal, request: "GET /api/users?page=1&limit=10&search= HTTP/1.1", upstream: "http://172.27.0.4:3000/api/users?page=1&limit=10&search=", host: "testlocal"
2025/05/09 18:35:30 [error] 282#282: *1611 connect() failed (111: Connection refused) while connecting to upstream, client: 172.27.0.1, server: testlocal, request: "GET /api/users?page=1&limit=10&search= HTTP/1.1", upstream: "http://172.27.0.4:3000/api/users?page=1&limit=10&search=", host: "testlocal"
2025/05/09 18:35:30 [error] 282#282: *1611 connect() failed (111: Connection refused) while connecting to upstream, client: 172.27.0.1, server: testlocal, request: "GET /api/users?page=1&limit=10&search= HTTP/1.1", upstream: "http://172.27.0.4:3000/api/users?page=1&limit=10&search=", host: "testlocal"
2025/05/09 18:35:30 [error] 282#282: *1611 connect() failed (111: Connection refused) while connecting to upstream, client: 172.27.0.1, server: testlocal, request: "GET /api/users?page=1&limit=10&search= HTTP/1.1", upstream: "http://172.27.0.4:3000/api/users?page=1&limit=10&search=", host: "testlocal"
2025/05/09 18:35:31 [error] 282#282: *1611 connect() failed (111: Connection refused) while connecting to upstream, client: 172.27.0.1, server: testlocal, request: "GET /api/users?page=1&limit=10&search= HTTP/1.1", upstream: "http://172.27.0.4:3000/api/users?page=1&limit=10&search=", host: "testlocal"
2025/05/09 18:35:31 [error] 282#282: *1611 connect() failed (111: Connection refused) while connecting to upstream, client: 172.27.0.1, server: testlocal, request: "GET /api/users?page=1&limit=10&search= HTTP/1.1", upstream: "http://172.27.0.4:3000/api/users?page=1&limit=10&search=", host: "testlocal"
2025/05/09 18:35:31 [error] 282#282: *1611 connect() failed (111: Connection refused) while connecting to upstream, client: 172.27.0.1, server: testlocal, request: "GET /api/users?page=1&limit=10&search= HTTP/1.1", upstream: "http://172.27.0.4:3000/api/users?page=1&limit=10&search=", host: "testlocal"
2025/05/09 18:35:42 [error] 188#188: *1 frontend could not be resolved (3: Host not found), client: 172.27.0.1, server: testlocal, request: "GET /users/list HTTP/1.1", host: "testlocal", referrer: "http://testlocal:81/"
2025/05/09 18:37:25 [error] 189#189: *14 connect() failed (111: Connection refused) while connecting to upstream, client: 172.27.0.1, server: testlocal, request: "GET /api/users?page=1&limit=10&search= HTTP/1.1", upstream: "http://172.27.0.4:3000/api/users?page=1&limit=10&search=", host: "testlocal", referrer: "http://testlocal/users/list"
2025/05/09 18:37:26 [error] 189#189: *14 connect() failed (111: Connection refused) while connecting to upstream, client: 172.27.0.1, server: testlocal, request: "GET /api/users?page=1&limit=10&search= HTTP/1.1", upstream: "http://172.27.0.4:3000/api/users?page=1&limit=10&search=", host: "testlocal", referrer: "http://testlocal/users/list"
2025/05/09 18:37:36 [error] 189#189: *14 connect() failed (111: Connection refused) while connecting to upstream, client: 172.27.0.1, server: testlocal, request: "GET /api/users?page=1&limit=10&search= HTTP/1.1", upstream: "http://172.27.0.4:3000/api/users?page=1&limit=10&search=", host: "testlocal", referrer: "http://testlocal/users/list"
2025/05/09 18:37:37 [error] 189#189: *14 connect() failed (111: Connection refused) while connecting to upstream, client: 172.27.0.1, server: testlocal, request: "GET /api/users?page=1&limit=10&search= HTTP/1.1", upstream: "http://172.27.0.4:3000/api/users?page=1&limit=10&search=", host: "testlocal"
2025/05/09 18:37:48 [error] 217#217: *28 connect() failed (111: Connection refused) while connecting to upstream, client: 172.27.0.1, server: testlocal, request: "GET /api/users?page=1&limit=10&search= HTTP/1.1", upstream: "http://172.27.0.4:9999/api/users?page=1&limit=10&search=", host: "testlocal", referrer: "http://testlocal/users/list"
2025/05/09 18:37:49 [error] 217#217: *28 connect() failed (111: Connection refused) while connecting to upstream, client: 172.27.0.1, server: testlocal, request: "GET /api/users?page=1&limit=10&search= HTTP/1.1", upstream: "http://172.27.0.4:9999/api/users?page=1&limit=10&search=", host: "testlocal"
2025/05/09 18:37:49 [error] 217#217: *28 connect() failed (111: Connection refused) while connecting to upstream, client: 172.27.0.1, server: testlocal, request: "GET /api/users?page=1&limit=10&search= HTTP/1.1", upstream: "http://172.27.0.4:9999/api/users?page=1&limit=10&search=", host: "testlocal"
2025/05/09 18:37:50 [error] 217#217: *28 connect() failed (111: Connection refused) while connecting to upstream, client: 172.27.0.1, server: testlocal, request: "GET /api/users?page=1&limit=10&search= HTTP/1.1", upstream: "http://172.27.0.4:9999/api/users?page=1&limit=10&search=", host: "testlocal"
2025/05/09 18:37:50 [error] 217#217: *28 connect() failed (111: Connection refused) while connecting to upstream, client: 172.27.0.1, server: testlocal, request: "GET /api/users?page=1&limit=10&search= HTTP/1.1", upstream: "http://172.27.0.4:9999/api/users?page=1&limit=10&search=", host: "testlocal"
2025/05/09 18:37:50 [error] 217#217: *28 connect() failed (111: Connection refused) while connecting to upstream, client: 172.27.0.1, server: testlocal, request: "GET /api/users?page=1&limit=10&search= HTTP/1.1", upstream: "http://172.27.0.4:9999/api/users?page=1&limit=10&search=", host: "testlocal"
2025/05/09 18:37:50 [error] 217#217: *28 connect() failed (111: Connection refused) while connecting to upstream, client: 172.27.0.1, server: testlocal, request: "GET /api/users?page=1&limit=10&search= HTTP/1.1", upstream: "http://172.27.0.4:9999/api/users?page=1&limit=10&search=", host: "testlocal"
2025/05/09 18:37:50 [error] 217#217: *28 connect() failed (111: Connection refused) while connecting to upstream, client: 172.27.0.1, server: testlocal, request: "GET /api/users?page=1&limit=10&search= HTTP/1.1", upstream: "http://172.27.0.4:9999/api/users?page=1&limit=10&search=", host: "testlocal"
2025/05/09 18:37:51 [error] 217#217: *28 connect() failed (111: Connection refused) while connecting to upstream, client: 172.27.0.1, server: testlocal, request: "GET /api/users?page=1&limit=10&search= HTTP/1.1", upstream: "http://172.27.0.4:9999/api/users?page=1&limit=10&search=", host: "testlocal"
2025/05/09 18:37:51 [error] 217#217: *28 connect() failed (111: Connection refused) while connecting to upstream, client: 172.27.0.1, server: testlocal, request: "GET /api/users?page=1&limit=10&search= HTTP/1.1", upstream: "http://172.27.0.4:9999/api/users?page=1&limit=10&search=", host: "testlocal"
2025/05/09 18:37:52 [error] 217#217: *28 connect() failed (111: Connection refused) while connecting to upstream, client: 172.27.0.1, server: testlocal, request: "GET /api/users?page=1&limit=10&search= HTTP/1.1", upstream: "http://172.27.0.4:9999/api/users?page=1&limit=10&search=", host: "testlocal", referrer: "http://testlocal/users/list"
2025/05/09 18:37:52 [error] 217#217: *28 connect() failed (111: Connection refused) while connecting to upstream, client: 172.27.0.1, server: testlocal, request: "GET /api/users?page=1&limit=10&search= HTTP/1.1", upstream: "http://172.27.0.4:9999/api/users?page=1&limit=10&search=", host: "testlocal"
2025/05/09 18:37:52 [error] 217#217: *28 connect() failed (111: Connection refused) while connecting to upstream, client: 172.27.0.1, server: testlocal, request: "GET /api/users?page=1&limit=10&search= HTTP/1.1", upstream: "http://172.27.0.4:9999/api/users?page=1&limit=10&search=", host: "testlocal"
2025/05/09 18:37:52 [error] 217#217: *28 connect() failed (111: Connection refused) while connecting to upstream, client: 172.27.0.1, server: testlocal, request: "GET /api/users?page=1&limit=10&search= HTTP/1.1", upstream: "http://172.27.0.4:9999/api/users?page=1&limit=10&search=", host: "testlocal"
2025/05/09 18:37:52 [error] 217#217: *28 connect() failed (111: Connection refused) while connecting to upstream, client: 172.27.0.1, server: testlocal, request: "GET /api/users?page=1&limit=10&search= HTTP/1.1", upstream: "http://172.27.0.4:9999/api/users?page=1&limit=10&search=", host: "testlocal"
2025/05/09 18:37:53 [error] 217#217: *28 connect() failed (111: Connection refused) while connecting to upstream, client: 172.27.0.1, server: testlocal, request: "GET /api/users?page=1&limit=10&search= HTTP/1.1", upstream: "http://172.27.0.4:9999/api/users?page=1&limit=10&search=", host: "testlocal", referrer: "http://testlocal/users/list"
2025/05/09 18:37:54 [error] 217#217: *28 connect() failed (111: Connection refused) while connecting to upstream, client: 172.27.0.1, server: testlocal, request: "GET /api/users?page=1&limit=10&search= HTTP/1.1", upstream: "http://172.27.0.4:9999/api/users?page=1&limit=10&search=", host: "testlocal", referrer: "http://testlocal/users/list"
2025/05/09 18:38:03 [error] 217#217: *28 connect() failed (111: Connection refused) while connecting to upstream, client: 172.27.0.1, server: testlocal, request: "GET /api/users?page=1&limit=10&search= HTTP/1.1", upstream: "http://172.27.0.4:9999/api/users?page=1&limit=10&search=", host: "testlocal", referrer: "http://testlocal/users/list"
2025/05/09 18:38:04 [error] 217#217: *28 connect() failed (111: Connection refused) while connecting to upstream, client: 172.27.0.1, server: testlocal, request: "GET /api/users?page=1&limit=10&search= HTTP/1.1", upstream: "http://172.27.0.4:9999/api/users?page=1&limit=10&search=", host: "testlocal", referrer: "http://testlocal/users/list"
2025/05/09 18:38:14 [error] 217#217: *28 connect() failed (111: Connection refused) while connecting to upstream, client: 172.27.0.1, server: testlocal, request: "GET /api/users?page=1&limit=10&search= HTTP/1.1", upstream: "http://172.27.0.4:9999/api/users?page=1&limit=10&search=", host: "testlocal", referrer: "http://testlocal/users/list"
2025/05/09 18:38:15 [error] 217#217: *28 connect() failed (111: Connection refused) while connecting to upstream, client: 172.27.0.1, server: testlocal, request: "GET /api/users?page=1&limit=10&search= HTTP/1.1", upstream: "http://172.27.0.4:9999/api/users?page=1&limit=10&search=", host: "testlocal", referrer: "http://testlocal/users/list"
